### DISCUSSION TITLE
Welcome to Chakra UI Discussions!

### DISCUSSION THREAD

**Discussion by segunadebayo** [chakra-ui/chakra-ui#1914] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by with-heart** [comment]
Howdy yall! 👋 

My name is Mark. I'm a React core team member from Cincinnati, Ohio.

I work for [VNDLY](https://vndly.com/), a startup based here in Cincinnati. I'm a developer embedded with our UX team, and I spend a lot of my work time building the component library for our app. Our decision to use Chakra as the basis for our component library is what initially engaged me with the project, and over time I've fallen in love with the library and the Chakra community, and now spend a lot of my free time working to help Chakra grow. 🌱

As a member of the React core team, I do a little bit of everything: code, repo management, pipeline/infrastructure, issue management, community engagement. I can be a little scatterbrained 🤯 so moving around helps me stay focused and engaged. I love solving problems and teaching and Chakra has been an excellent way for me to express both of those loves.

I'd like to end by saying that we're glad you're here! If there's anything we can do to help you on your Chakra journey, please don't hesitate to reach out here in the discussions, through issues if you have bug reports or feature requests, or through the [official Chakra Discord](https://discord.gg/dQHfcWF)!

**Comment by harveyhalwin** [comment]


**Reply by jmiazga** [reply]
Hey @harveyhalwin. Any chance you can move this to its own discussion?

**Comment by aaronmcadam** [comment]
Hi everyone! 👋

My name is Aaron. I'm a UX Engineer from Ireland and living in Leicester, UK.

I work for a mobile games company called [Lockwood Publishing](https://lockwoodpublishing.com/). I work on tools to help people create new content for our game, [Avakin Life](https://avakin.com/).

I've been using and loving Chakra for about a year now for building my design systems and features, and it's been fantastic! I've used other tools, such as `styled-system` with `rebass` in the past. Chakra improves upon their ideas so much!

Because of my experience with Chakra, I've been trying to help out here and there where I can. I try to answer a few questions when I can.

I'm really looking forward to v1 because it seems like a great release! I think it introduces some exciting patterns based on lessons learned from real-world use cases.

I'd like to thank @segunadebayo and @with-heart and everyone else involved in this awesome community! ❤️

**Comment by claudiofsn** [comment]
Hi guys!

I'm having a small issue when using chakra-ui/react + imask + TypeScript. I'm building the following component:
const MaskedStyledInput = IMaskMixin(({ inputRef, ...props }) => (
    <Input as="input" {...props} ref={inputRef} />
))

But TypeScript is showing the following error message:

![image](https://github.com/chakra-ui/chakra-ui/assets/88662893/d539587d-7acd-46bb-b12e-bc5ecacc2740)

Do you know how I can solve it?
